mmv1/products/securesourcemanager/Instance.yaml

```diff
@@ -85,6 +85,41 @@ examples:
       instance_id: 'my-instance'
       ca_pool_id: 'ca-pool'
       root_ca_id: 'root-ca'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'secure_source_manager_instance_private_psc_backend'
+    external_providers: ["random", "time"]
```

Comment:
Nit: I know this was probably copied from the others, but you aren't using the `random` provider (from what I can tell), so it should probably be removed.

Reply:
Removed. The previous was added by myself actually, although I can't remember why I added it...